components/camel-openapi-java/src/main/java/org/apache/camel/openapi/RestOpenApiSupport.java

```diff
@@ -60,7 +61,12 @@
  */
 public class RestOpenApiSupport {
 
-    public static DateFormat DEFAULT_DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd");
+    private static final DateFormat DEFAULT_DATE_FORMAT;
+    static {
+        final SimpleDateFormat f = new SimpleDateFormat("yyyy-MM-dd");
+        f.setTimeZone(TimeZone.getTimeZone("UTC"));
```

Comment:
In the original test, it used `testDateFormat.setTimeZone(TimeZone.getTimeZone("GMT"));`, not sure if it is the same with `"UTC"`?